internal/pkg/ecs/ecs.go

```diff
@@ -465,3 +466,41 @@ func (c Client) stateMachineARN(app, env, job string) (string, error) {
 	}
 	return stateMachineARN, nil
 }
+
+// HasNonZeroExitCode returns an error if at least one of the tasks exited with a non-zero exit code. It assumes that all tasks are built on the same task definition.
+func (c Client) HasNonZeroExitCode(taskARNs []string, cluster string) error {
+	tasks, err := c.ecsClient.DescribeTasks(cluster, taskARNs)
+	if err != nil {
+		return fmt.Errorf("describe tasks %s: %w", taskARNs, err)
+	}
+
+	if len(tasks) == 0 {
+		return fmt.Errorf("describe tasks: none of the tasks %s exist", strings.Join(taskARNs, ","))
```

Comment:
nit: this is not an error returned by `DescribeTasks`.
```suggestion
		return fmt.Errorf("cannot find tasks %s", strings.Join(taskARNs, ", "))
```